components/dialog/test/dialog.vdiff.js

```diff
@@ -29,7 +29,7 @@ describe('dialog', () => {
 			[
 				{ screen: 'tall-wide', viewport: { width: 800, height: 500 } },
 				{ screen: 'tall-narrow', viewport: { width: 600, height: 500 } },
-				{ screen: 'short-wide', viewport: { width: 910, height: 400 } },
+				{ screen: 'short-wide', viewport: { width: 910, height: 370 } },
```

Comment:
I adjusted the height of the dialog so that the screenshot would capture the shadow like it previously did. Though I don't think that's especially important here, since the shadows are captured more explicitly in a couple other diffs.